src/assets/css/ProfilePage.css

```
@@ -0,0 +1,64 @@
+.container {
+    display: flex;
+}
+
+.some-page-wrapper {
+    display: flex;  
+    flex-direction: column;
+    width: 100%;
+    
+}
+
+.column{
+    display: flex;
+    height: 607px;
+    width: 784px;  
+    align-self: center;
+    border: 1px solid white;  
+    border-radius: 25px;
+    flex-wrap: wrap;
+    
+    
+}
+
+.Save-button {
+    margin-left: 41%;
+    width: 100%;
+    align-items: center;
+    height:100px;
+    
+}
+.button {
+    background-color: #11426B;
+    color: white;
+    align-self: center;
+    width: 135px;
+    height: 55px;
+    border-radius: 30px;
+}
+
+.one-column {
+    height: 450px;
+    width: 390px;
+    border: 1px solid white;
```

```suggestion
    border: 1px solid white;
```
Remove border radius